From 497446658a71d2373a43b4f74aff6b1c5b8f57cd Mon Sep 17 00:00:00 2001
From: Mitsutoshi Aoe <me@maoe.name>
Date: Tue, 23 Feb 2021 17:21:07 +0900
Subject: [PATCH] Fix a parse error to make it buildable with GHC 9.0.1
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The error message was:

  GUI/SaveAs.hs:30:37: error: parse error on input ‘{’
     |
  30 | saveAs hecs params' @ViewParameters { xScaleAreaHeight, width,
     |                                     ^
---
 GUI/SaveAs.hs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/GUI/SaveAs.hs b/GUI/SaveAs.hs
index 40f772a..a576b5d 100644
--- a/GUI/SaveAs.hs
+++ b/GUI/SaveAs.hs
@@ -27,7 +27,7 @@ import Graphics.Rendering.Cairo
 
 saveAs :: HECs -> ViewParameters -> Double -> DrawingArea
        -> (Int, Int, Render ())
-saveAs hecs params' @ViewParameters{xScaleAreaHeight, width,
+saveAs hecs params'@ViewParameters{xScaleAreaHeight, width,
                                     height = oldHeight {-, histogramHeight-}}
        yScaleAreaWidth yScaleArea =
   let histTotalHeight = histXScaleHeight -- + histogramHeight
